Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml

```diff
@@ -13,11 +13,15 @@
     size: 338, 274
     setup:
       !type:TabletopCheckerSetup
-      boardPrototype: CheckerBoardTabletop
+      boardPrototype: &checkerboard CheckerBoardTabletop
+      prototypePieceWhite: &pieceWhite CheckerPieceWhite
+      prototypeCrownWhite: &crownWhite CheckerCrownWhite
+      prototypePieceBlack: &pieceBlack CheckerPieceBlack
+      prototypeCrownBlack: &crownBlack CheckerCrownBlack
 
 # Checkerboard tabletop item (item only visible in tabletop game)
 - type: entity
-  id: CheckerBoardTabletop
+  id: *checkerboard
```

Comment:
Why are the prototype IDs being renamed? Please don't change prototype IDs unless there is a very good reason to do so. Also make sure the new names follow the prototype naming conventions.

Reply:
`*checkboard` doesn't follow ID naming conventions I believe

Reply:
> because it's not an id. [it's an alias of an id](https://github.com/space-wizards/space-station-14/blob/c898dcdd25c25be82172b4511ada6547da5f5de5/Resources/Prototypes/Entities/Objects/Fun/Tabletop/checkers.yml#L16C23-L16C57).

oh, my bad lol

Reply:
They're used extremely sparingly in ss14 but they are valid yaml.